index.d.ts

```diff
@@ -0,0 +1,30 @@
+// Type definitions for eris-pagination v0.2.2
+// Project: eris-pagination
+// Author: riyacchi <https://github.com/riyacchi/>
+// Definitions by: CircuitRCAY <https://github.com/CircuitCodes>
+
+declare module 'eris-pagination' {
+    import { Message, EmbedOptions } from 'eris';
+
+    class PaginationEmbed {
+        constructor(message: Message, pages: EmbedOptions[], options?: PaginationEmbedOptions);
+        public initialize(): Promise<void>;
+        public run(): void;
+        public update(): void;
+    }
```

Comment:
This class definition isn't necessary since it isn't accessible to API consumers. It should be removed.